sql/updates/world/fix black_temple Teleporter.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE creature_template SET gossip_menu_id=0,Scriptname = "npc_Spirit_of_Udalo" WHERE entry=23410;
```

Comment:
This file does not follow SQL guidelines http://collab.kpsn.org/display/tc/SQL+Guidelines